oshmem/mca/spml/ucx/spml_ucx_component.c

```diff
@@ -101,6 +101,10 @@ static int mca_spml_ucx_component_register(void)
                                     "How may disconnects go in parallel",
                                     &mca_spml_ucx.num_disconnect);
 
+    mca_spml_ucx_param_register_int("heap_reg_nb", 0,
```

Comment:
What happens if the flag is not supported for the device? Will ucp_mem_map() fail ?